model/src/main/java/uk/gov/hmcts/payment/api/service/UserAwareDelegatingCardPaymentService.java

```diff
@@ -50,6 +52,11 @@
     private final ServiceIdSupplier serviceIdSupplier;
     private final AuditRepository auditRepository;
 
+    @Autowired
+    private Environment environment;
+
+    private String[] testProfiles = {"embedded", "local", "componenttest"};
```

Comment:
should there be test specific logic? would that defeat the whole test logic and flow?